index.html

```diff
@@ -10,7 +10,8 @@
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
-
```

Comment:
<--! Would it be a good idea to re-name the favicon so that it is descriptive of what the image is? -->